src/urlValidation/urlValidation.js

```diff
@@ -0,0 +1,31 @@
+function urlValidation(url) {
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const [textToConvert, searchParams] = url.split('?');
+
+  const errors = [];
+  const params = new URLSearchParams(searchParams);
+  const toCase = params.get('toCase');
+  const normilizedText = textToConvert.slice(1);
+
+  if (textToConvert.length < 2) {
```

Comment:
What does 2 in this case? Magic number - use constants.

Reply:
thank you!)